src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php

```diff
@@ -123,6 +123,33 @@ private function getFileKey(string $file): string
         return '';
     }
 
+    private function scanHashDir(string $directory): \Generator
+    {
+        if (!file_exists($directory)) {
+            return;
+        }
+
+        $chars = '+-ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
```

Comment:
I guess you'll disagree 😝  but I'd define a `$charsLength = 38;` variable (or, ideally, `$charsLength = \strlen($chars);`) to avoid repeating this "magic number" in the code. Thanks.

Reply:
you've been right :) this is purely internal, adding an indirection for the very next line is not worth it.